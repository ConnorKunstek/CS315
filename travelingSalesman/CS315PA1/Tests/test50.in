Archidiaconate 36.3782 -84.0296
Prophetic 36.4016 -75.3438
Waylay 33.7106 -76.6118
Demiram 32.9735 -88.2774
Canonry 38.3321 -89.0699
Landuman 32.7395 -78.07
Unfinishedness 36.4484 -73.315
Hornlike 33.4181 -87.0411
Tret 34.6115 -74.0758
Superstatesman 32.9618 -87.2313
Cerolite 35.9453 -87.7385
Catalogue 33.9914 -75.9461
Hulk 36.7175 -79.3063
Suspicional 37.0919 -86.8509
Molybdomenite 37.9343 -78.6406
Cod 36.1442 -77.0556
Quinology 35.9219 -80.2256
Greenly 34.9742 -79.8135
Neckatee 36.3197 -73.9173
Lurchline 37.3025 -88.1823
Intropulsive 34.4477 -73.5369
Crambambuli 38.3906 -80.0354
Undiminishable 32.7746 -75.0902
Perissodactylous 39.0809 -87.6751
Worble 33.7925 -89.1016
Zoografting 34.3424 -78.9259
Sphingomyelin 32.3768 -78.1651
Dark 38.6246 -80.1622
Dexterously 35.0912 -77.3092
Dendrocoelous 32.5991 -82.0008
Seleniuret 32.4938 -76.4533
Spatting 36.3899 -73.1248
Trisinuate 33.5702 -84.6319
Unqualify 33.4649 -78.1968
Coprophagous 35.0561 -82.7616
Indistinctiveness 37.2791 -85.8682
Groundmass 36.7877 -73.1882
Enchequer 32.6693 -84.727
Corollarial 34.1201 -81.5253
Wodge 38.3087 -76.0095
Sialagogic 32.1779 -79.6867
Careerist 35.2784 -76.485
Hiller 39.1628 -83.5541
Solver 39.2915 -81.3351
Graphometry 34.3658 -87.5483
Messianic 32.3417 -80.6377
Boobily 33.9446 -82.6665
Hapuku 33.7457 -87.897
Pleiochromic 37.2674 -76.0412
Misadvisedness 32.8916 -81.9691
