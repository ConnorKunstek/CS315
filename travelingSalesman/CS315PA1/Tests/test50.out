

West to East: Worble to Spatting
Distance in Kilometers: 14480 km
Distance in Miles: 8997 miles
North to South: Solver to Sialagogic
Distance in Kilometers: 25379 km
Distance in Miles: 15769 miles
Time: 0.000904 seconds

